primap2/tests/test_data_reading.py

```diff
@@ -151,10 +152,10 @@ def test_convert_entity_gwp_primap(entity_pm1, entity_pm2):
 
 
 def test_read_wide_csv_file():
-    path = os.path.join("primap2", "tests", "data")
-    file_input = "test_csv_data_sec_cat.csv"
-    file_expected = "test_read_wide_csv_file_output.csv"
-    df_expected = pd.read_csv(os.path.join(path, file_expected), index_col=0)
+    path = Path("primap2") / Path("tests") / Path("data")
```

Comment:
You can even write this:
```python
path = Path("primap2") / "tests" / "data"
```
by operator overloading magic, a path "divided by" a string is a path again.

Reply:
I'll use that for new code and changed most old code.